pilot/pkg/serviceregistry/kube/controller/multicluster_test.go

```diff
@@ -34,6 +34,7 @@ import (
 const (
 	testSecretName      = "testSecretName"
 	testSecretNameSpace = "istio-system"
+	testRevision        = "asm-168-3"
```

Comment:
should use a different label probably :slightly_smiling_face:

Reply:
whoops, forgot where I was for a second